### PR TITLE
Small improvements to docs for version command

### PR DESCRIPTION
**Pull request by iliakur**
This is just docs, the template didn't seem to apply.

### CODE REVIEW

File: docs/docs/cli.md [hunk 1]
```diff
@@ -428,13 +428,27 @@ This command shows the current version of the project or bumps the version of
...
 
-## Options
+The table below illustrates the effect of these rules with concrete examples.
+
+| rule       |        before | after         |
+|------------|---------------|---------------|
+| major      |         1.3.0 | 2.0.0         |
+| minor      |         2.1.4 | 2.2.0         |
+| patch      |         4.1.1 | 4.1.2         |
+| premajor   |         1.0.2 | 2.0.2-alpha.0 |
```
Comment (finswimmer) [inline docs/docs/cli.md:441]: This should become `2.0.0-alpha.0

Comment (iliakur) [reply docs/docs/cli.md]: Good catch!

**Comment by iliakur** [comment]
Thanks for the feedback, added some more examples :)

**Comment by iliakur** [comment]
No problem, do you want me to change the base branch directly here in github or cherry-pick/rebase locally and then open a new PR?

**Comment by finswimmer** [comment]
@iliakur you should rebase on master first and then change the base branch here on github. There should be no need to open a new PR.

**Comment by iliakur** [comment]
@finswimmer rebasing was trickier than I thought and also required that I force-push.

I would be willing to add a sentence or two into `CONTRIBUTING.md` that tell folks which branch to base their work on depending on whether they are contributing code or documentation/tests that don't affect existing functionality.

Or do you think this is obvious to most people and I'm juts a n00b? That's fine, I'm a bit new to contributing to bigger projects ;)

**Comment by iliakur** [comment]
Bump @finswimmer  :)

**Comment by finswimmer** [comment]
Hello @iliakur,

for some reasons the checks doesn't start. So I'm unable to merge. Can you submit a very minor change to force the checks to restart?

fin swimmer

**Comment by iliakur** [comment]
CI seems to have passed :tada:

**Pull request merged**